resources/css/main.css

```diff
@@ -1,37 +1,499 @@
+  margin: 0 auto;
+  margin-top: 15px;
+}
+
+.inflation-header-title {
+  font-family: georgia, "times new roman", serif;
+  font-style: normal;
+  font-size: 17px;
+  font-weight: 700;
+  margin-bottom: 2px;
+}
+
+.inflation-header-subtitle {
+  color: rgb(102, 102, 102);
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.inflation-container {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 15px;
+}
+
+.inflation-container-section {
+  width: 25%;
+  padding-right: 15px;
+  padding-bottom: 15px;
+}
+
+.inflation-container-section-text {
+  font-family: arial, helvetica, sans-serif;
+  font-size: 13px;
+  font-weight: 300;
+  font-style: normal;
+  line-height: 17px;
+  color: black;
+}
+
+.inflation-container-section-first-phrase {
+  font-family: inherit;
+  font-size: 11px;
+  font-weight: bold;
+  letter-spacing: 0.05em;
+}
+
+.inflation-footer::before {
+  margin-bottom: 13px;
+  background-color: black;
+  display: inline-block;
+  width: 60px;
+  height: 1px;
+  content: "";
+}
+
+.inflation-footer-authors {
+  color: rgb(51, 51, 51);
+  font-family: arial, helvetica, sans-serif;
+  font-size: 13px;
+}
+
+/** Site index **/
+#site-index {
+  font-family: Arial, Helvetica, sans-serif;
+  width: 1200px;
+  margin: 90px auto 10px;
+  border-top: 1px solid #e2e2e2;
+}
+
+#site-index::before {
+  content: "";
+  height: 2px;
+  background-color: #e2e2e2;
+  width: 100%;
+  display: block;
+  position: relative;
+  top: -4px;
+}
+
+.site-index-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.link-blue {
+  color: rgb(86, 123, 139);
+  text-decoration: none;
+  font-size: 11px;
+}
+
+.site-index-logo {
+  width: 185px;
+  display: inline-block;
+  margin-bottom: 20px;
+  margin-top: 10px;
+}
+
+.site-index-container {
+  display: grid;
+  grid-template-columns: 16% 16% 16% 16% 16% 20%;
+}
+
+.site-index-section--highlight {
+  border-left: 1px solid #e2e2e2;
+  padding-left: 20px;
+}
+
+.site-index-section-label {
+  color: black;
+  font-size: 13px;
+  text-transform: uppercase;
+  display: inline-block;
+  margin-bottom: 6px;
+}
+
+.site-index-section-list {
+  list-style: none;
+}
+
+.site-index-section-list--highlight {
+  list-style: none;
+  margin-bottom: 15px;
+}
+
+.site-index-section-list-item {
+  font-size: 14px;
+  padding: 4px 0;
+  cursor: pointer;
+}
+
+.site-index-section-list-item:hover {
+  text-decoration: underline;
+}
+
+.site-index-section-list--highlightsite-index-section-list-item {
+  font-weight: 700;
+  font-size: 16px;
+  margin-bottom: 5px;
+}
+
+.site-index-section-list--small {
+  margin-bottom: 10px;
+  padding-bottom: 6px;
+}
+
+.site-index-section-list--small .site-index-section-list-item {
+  font-size: 11px;
+  padding-bottom: 6px;
+}
+
+.site-index-section-list-item .icon {
+  width: 12px;
+}
+
+.border-bottom {
+  border-bottom: 1px solid #e2e2e2;
+}
+
+#site-footer {
```

Comment:
- [ ] Don't use IDs in selectors.

Reply:
We fixed this by using a class instead. Thank you!